content/master/concepts/providers.md

```diff
@@ -3,52 +3,306 @@ title: Providers
+resources must be manually deleted by removing their finalizers. 
+
+For more information on deleting abandoned resources read the [Crossplane
+troubleshooting guide]({{<ref "/knowledge-base/guides/troubleshoot#deleting-when-a-resource-hangs" >}}). 
+{{< /hint >}}
+
+## Verify a Provider
+
+Providers install their own APIs representing the managed resources they support.
+Providers may also create Deployments, Service Accounts or RBAC configuration. 
+
+View the status of a Provider with 
+
+`kubectl get providers`
+
+During the install a Provider report `INSTALLED` as `True` and `HEALTHY` as
+`Unknown`.
+
+```shell
+kubectl get providers
+NAME                              INSTALLED   HEALTHY   PACKAGE                                                   AGE
+crossplane-contrib-provider-aws   True        Unknown   xpkg.upbound.io/crossplane-contrib/provider-aws:v0.39.0   63s
 ```
 
-The field `spec.package` is where you refer to the container image of the
-provider. Crossplane Package Manager will unpack that container, register CRDs
-and set up necessary RBAC rules and then start the controllers.
+After the Provider install completes and it's ready for use the `HEALTHY` status
+reports `True`.
 
-There are a few other ways to to trigger the installation of provider packages:
+```shell
+kubectl get providers
+NAME                              INSTALLED   HEALTHY   PACKAGE                                                   AGE
+crossplane-contrib-provider-aws   True        True      xpkg.upbound.io/crossplane-contrib/provider-aws:v0.39.0   88s
+```
 
-* As part of Crossplane Helm chart by adding the following statement to your
-  `helm install` command: `--set
-  provider.packages={xpkg.upbound.io/crossplane-contrib/provider-aws:v0.33.0}`.
-* Using the Crossplane CLI: `kubectl crossplane install provider
-  xpkg.upbound.io/crossplane-contrib/provider-aws:v0.33.0`
+{{<hint "important" >}}
+Some Providers install hundreds of Kubernetes Custom Resource Definitions (`CRDs`).
+This can create significant strain on undersized API Servers, impacting Provider
+install times. 
 
-You can uninstall a provider by deleting the `Provider` resource
-you've created.
+The Crossplane community has more 
+[details on scaling CRDs](https://github.com/crossplane/crossplane/blob/master/design/one-pager-crd-scaling.md).
+{{< /hint >}}
+### Provider conditions
 
-## Configuring Providers
+View the conditions of a provider under their `Status` with 
+`kubectl describe provider`.
 
-In order to authenticate with the external provider API, the provider
-controllers need to have access to credentials. It could be an IAM User for AWS,
-a Service Account for GCP or a Service Principal for Azure. Every provider has a
-type called `ProviderConfig` that has information about how to authenticate to
-the provider API. An example `ProviderConfig` resource for AWS looks like the
-following:
+Providers have the following possible conditions:
```

Comment:
Nit: Technically the condition is the type (i.e.) `Installed`. Things like `ActivePackageRevision` are the reason that the condition is true or false.

Reply:
I'm using "condition" to refer to the set of {Type, Status, Reason}. Specifically `Condition` coming from the k8s output. 

Yes, the headings are the `Reason` but I wanted to key on the thing that would be unique to make it easier for readers to find what they're looking for.